ui/app/AppLayouts/Chat/views/MembersSelectorView.qml

```diff
@@ -107,7 +107,7 @@ MembersSelectorBase {
                 return
             }
 
-            if (root.model.count === 0) {
+            if (root.model.count === 0 && !root.rootStore.contactsStore.hasPendingContactRequest(contactDetails.publicKey)) {
```

Comment:
In that case, we should popup information that the contact request has been already sent to the user.

Reply:
I added a link on another issue about it :)